content/pt/docs/concepts/observability-primer.md

```diff
@@ -0,0 +1,156 @@
+funcionamento interno. Além disso, facilita a resolução de problemas e o
+tratamento de novos problemas, ou seja, "desconhecidos desconhecido". Também
+ajuda a responder à pergunta "Por que isso está acontecendo?"
+
+Para fazer essas perguntas sobre o seu sistema, sua aplicação deve estar
+devidamente instrumentada. Ou seja, o código da aplicação deve emitir
+[sinais](/docs/concepts/signals/) como
+[rastros](/docs/concepts/signals/traces/),
+[métricas](/docs/concepts/signals/metrics/) e
+[logs](/docs/concepts/signals/logs/). Uma aplicação está devidamente
+instrumentada quando os desenvolvedores não precisam adicionar mais
+instrumentação para solucionar um problema, pois já tem todas as informações
+necessárias.
+
+O [OpenTelemetry](/docs/what-is-opentelemetry/) é o mecanismo pelo qual o código
+da aplicação é instrumentado para ajudar a tornar um sistema observável.
+
+## Confiabilidade e métricas
+
+**Telemetria** refere-se aos dados emitidos por um sistema e seu comportamento.
+Esses dados podem vir na forma de [rastros](/docs/concepts/signals/traces/),
+[métricas](/docs/concepts/signals/metrics/) e
+[logs](/docs/concepts/signals/logs/).
+
+**Confiabilidade** responde à perguntas: "O serviço está fazendo o que os
+usuários esperam que ele faça?" Um sistema pode estar ativo 100% do tempo, mas
+se, quando um usuário clica em "Adicionar ao Carrinho" para adicionar um par de
+sapatos pretos ao carrinho de compras, o sistema nem sempre adiciona sapatos
+pretos, então o sistema pode ser **não** confiável.
+
+**Métricas** são agregações ao longo de um período de tempo, de dados numéricos
+sobre sua infraestrutura ou aplicação. Exemplos incluem: taxa de erro do
+sistema, utilização da CPU e taxa de requisições para um determinado serviço.
+Para mais informações sobre métricas e como elas se relacionam com o
+OpenTelemetry, consulte [Métricas](/docs/concepts/signals/metrics/).
+
+**SLI**, ou _Service Level Indicator_, representa uma medida do comportamento de
+um serviço. Um bom SLI mede seu serviço do ponto de vista dos seus usuários. Um
+exemplo de SLI pode ser a velocidade de carregamento de uma página web.
+
+**SLO**, ou _Service Level Objective_, representa a forma como a confiabilidade
+é comunicada para uma organização/outras equipes. Isso é feito associando um ou
+mais SLIs ao valor de negócio.
+
+## Compreendendo rastreamento distribuído
+
+O rastreamento distribuído permite que você observe as requisições que se
+propagam por sistemas complexos e distribuídos. O rastreamento distribuído
+melhora a visibilidade da saúde da sua aplicação ou sistema, permitindo que você
+faça depurações de comportamentos difíceis de reproduzir localmente. É essencial
+para sistemas distribuídos, que comumente têm problemas não determinísticos ou
+são muito complicados para reproduzir localmente.
+
+Para entender o rastreamento distribuído, você precisa compreender a função de
+cada um de seus componentes: logs, trecho e rastros.
```

Comment:
Talvez a gente possa manter *span* no original ou traduzir e fazer uma referência, não sei se ficou muito bom *trecho*. O que acha?

Reply:
@edsoncelio acreditar que tinha colocado `spans`, fiquei com dúvida, não sei se fica legal traduzir trace e não traduzir span. 

Minha lógica foi, span não é um componente do projeto, então deve ser traduzido. Eu gostaria de usar os sinais sem tradução, trace, metric, log, profiling, etc. Dessa forma o público fica familiarizado com os termos.

Reply:
É, complicado.
Alguns desses termos já tão bem difundidos no ptbr, como métricas, rastros e outros nem tanto.
Eu sinceramente tô meio sem opinião nesse caso, ao mesmo tempo que seria bom ter traduzido a gente perde porque maioria so se usa o original mesmo.

Reply:
Se seguir sem tradução para os sinais, devemos subir um fix nos PR já "mergiados". Né?

Reply:
Isso, mas queria mais alguns feedbacks, tô realmente na dúvida hah
O que eu sugiro: segue do jeito que tá pra não bloquear o PR, e puxamos uma discussão, dai se o consenso for deixar o original a gente abre uma issue/pr pra alterar em tudo, o que acha?

Reply:
Concordo, meu voto é seguir com sinais sem tradução. Nas ferramentas de monitoramento/observabilidade esses termos é em inglês.

Reply:
Eu particularmente prefiro traduzir: rastros, trechos, métricas, perfis. Logs não tem jeito. O que eu gosto de fazer é usar em português e depois repetir em inglês na língua falada. Na escrita, eu colocaria o termo em inglês em parenteses quando do primeiro uso.

Eu acho que tem muito valor usar termos em português, as pessoas acabam tendo um contexto maior, facilitando o entendimento e removendo a "mística".

Reply:
Acho que pro contexto de uma documentação é ok usar trechos. Eu votaria por manter assim. Pode parecer estranho em um primeiro momento pq estamos acostumados a ler span, mas a medida q isso for difundido, a tendência é ficar normal de se ler.

Reply:
Por mim parece ótimo! Podemos seguir assim!

Reply:
OK, a maioria optou por seguir com a tradução dos sinais.